### PR TITLE
fix: MD033/no-inline-html

### PR DESCRIPTION
**Pull request by nschonni**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by mamccrea — APPROVED** [review]

File: docs/spark/tutorials/databricks-deployment.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Before you start, do the following:
 
 [Databricks](https://databricks.com) is a platform that provides cloud-based big data processing using Apache Spark.
 
-> [!Note] 
+> [!Note]
```
Comment (mairaw) [inline docs/spark/tutorials/databricks-deployment.md]:
```suggestion
> [!NOTE]
```

File: docs/standard/io/pipelines.md [hunk 2]
```diff
@@ -299,6 +300,7 @@ When writing helpers that read the buffer, any returned payload should be copied
 
 ## PipeWriter
 
+<!-- markdownlint-disable MD033 BUG with backticks in ref links -->
```
Comment (mairaw) [inline docs/standard/io/pipelines.md]:
```suggestion
```

File: docs/standard/io/pipelines.md [hunk 3]
```diff
@@ -299,6 +300,7 @@ When writing helpers that read the buffer, any returned payload should be copied
...
 The <xref:System.IO.Pipelines.PipeWriter> manages buffers for writing on the caller's behalf. `PipeWriter` implements [`IBufferWriter<byte>`](xref:System.Buffers.IBufferWriter`1). `IBufferWriter<byte>` makes it possible to get access to buffers to perform writes without additional buffer copies.
```
Comment (mairaw) [inline docs/standard/io/pipelines.md]: link shouldn't have backticks 

```suggestion
The <xref:System.IO.Pipelines.PipeWriter> manages buffers for writing on the caller's behalf. `PipeWriter` implements [`IBufferWriter<byte>`](xref:System.Buffers.IBufferWriter%601). `IBufferWriter<byte>` makes it possible to get access to buffers to perform writes without additional buffer copies.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM but left some suggestions

**Review by mairaw — APPROVED** [review]

**Pull request merged**